cmake/make_versioncpp.py

```diff
@@ -49,8 +46,9 @@
 version_cpp = open(outfile, "w")
 version_cpp.write(template.substitute(
     FreeOrion_VERSION=version,
-    FreeOrion_WC_REVISION=wc_rev,
+    FreeOrion_BRANCH=branch,
+    FreeOrion_BUILD_NO=build_no,
     FreeOrion_BUILDSYS=build_sys))
 version_cpp.close()
 
-print "Building rev: " + wc_rev
+print "Building v%s build %s" %(version, build_no)
```

Comment:
Git hub shows notification about no new line at the end of file. I ask Freeorion team no accept any pull request with this bug.